client/my-sites/stats/controller.jsx

```diff
@@ -305,7 +305,7 @@ export function summary( context, next ) {
 
 	// All Time Summary Support
 	if ( queryOptions.summarize && queryOptions.num ) {
-		statsQueryOptions = pick( queryOptions, [ 'num', 'summarize' ] );
+		statsQueryOptions = pick( queryOptions, [ 'num', 'summarize', 'geoMode' ] );
```

Comment:
The parameter `geoMode` does not seem relevant to the [All Time Summary Support](https://github.com/Automattic/wp-calypso/pull/103223/files#diff-71e9a204a637b410f5b0c4072fb0001b980183c95b390c1adfb92c1d8ea3df09R306). 🤔